posthog/tasks/calculate_cohort.py

```diff
@@ -72,6 +69,11 @@ def clear_stale_cohort(cohort_id: int, before_version: int) -> None:
 @shared_task(ignore_result=True, max_retries=2)
 def calculate_cohort_ch(cohort_id: int, pending_version: int, initiating_user_id: Optional[int] = None) -> None:
     cohort: Cohort = Cohort.objects.get(pk=cohort_id)
+
+    set_tag("feature", Feature.COHORT.value)
```

Comment:
`set_tag` call needs to be in the same thread (celery task) for sentry scoping to work